test/functional/test_framework/test_node.py

```
@@ -878,6 +878,9 @@ def __init__(self, rpc, cli=False, descriptors=False):
     def __getattr__(self, name):
         return getattr(self.rpc, name)
 
+    def createwallet_passthrough(self, *args, **kwargs):
```

Ruff (0.12.2)

Missing return type annotation for public function `createwallet_passthrough`

(ANN201)